eth/client.go

```diff
@@ -161,6 +162,9 @@ func NewClient(accountAddr ethcommon.Address, keystoreDir, password string, eth
 		return nil, err
 	}
 	backend.SetMaxGasPrice(maxGasPrice)
+	if monitor.Enabled && maxGasPrice != nil {
```

Comment:
Oh I see now that the only way to set the maxGasPrice field on `backend` is via the `SetMaxGasPrice()` method right now. And we're calling that method on L164 which would record the maxGasPrice. So, it actually doesn't seem like we need this piece of code - we'd only need to record the maxGasPrice in other areas besides within the `SetMaxGasPrice()` method if the maxGasPrice field can be set in other areas.